language_formatters_pre_commit_hooks/utils.py

```diff
@@ -90,17 +88,12 @@ def remove_trailing_whitespaces_and_set_new_line_ending(string: str) -> str:
 
 
 def does_checksum_match(path: str, expected: str) -> bool:
-    sha256 = hashlib.sha256()
-
-    with open(path, "rb") as f:
-        while True:
-            chunk = f.read(CHECKSUM_CHUNK_SIZE_KiB)
-            if not chunk:
-                break
-
-            sha256.update(chunk)
+    with open(path, "rb", buffering=0) as f:  # pragma: no cover
+        if sys.version_info >= (3, 11):
+            actual = hashlib.file_digest(f, "sha256").hexdigest()  # type: ignore[attr-defined]
+        else:
+            actual = hashlib.sha256(f.read()).hexdigest()
```

Comment:
Is there a specific reason to not use the same implementation independently from the Python version?

Whether possible I would aim for simple and unique implementation

Reply:
I guess my style is to be future-facing when possible, but I understand the desire to be consistent. Will update